compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Popup.skiko.kt

```diff
@@ -101,13 +101,11 @@ fun Popup(
     onKeyEvent: ((KeyEvent) -> Boolean) = { false },
     focusable: Boolean = false,
     content: @Composable () -> Unit
-) {
-    PopupLayout(
-        popupPositionProvider,
-        focusable,
-        onDismissRequest,
-        onPreviewKeyEvent,
-        onKeyEvent,
-        content
-    )
-}
+) = PopupLayout(
+    popupPositionProvider = popupPositionProvider,
+    focusable = focusable,
+    onClickOutside = if (focusable) onDismissRequest else null,
```

Comment:
When `focusable == false`, then `onClickOutside` is always null.
Why this behavior?
Maybe it is possible to close Popup even if it is not focusable?

For example, if this Popup can be used on iOS sourceSet, then we can't control its visibility with mouse hover. We would like to control it with touch events.

Reply:
Or maybe for iOS `fousable == false` have no sense?

Reply:
> Why this behavior?

Keep existing behaviour for existing overload

> Maybe it is possible to close Popup even if it is not focusable?

It's a separate flag in new commonized Popup